source/_posts/2024-09-30-call-fo-sponsors.md

```diff
@@ -0,0 +1,83 @@
+
+---
+title: 'The PHP Foundation Calls for Sponsors: Help&nbsp;Shape the Future of PHP'
```

Comment:
Using HTML entities breaks the RSS feed as it does not expect HTML in its title.

This should have use the Yaml `\_` escape sequence instead to add a non-breaking space in a plain-text string.

Reply:
Ah, thanks for noticing!
I tried escaping as you said, but it didn't work. So I fixed differently here #169